### PR TITLE
HS60 v2: Instructions how to reset and flash

### PR DESCRIPTION
**Pull request by andys8**
## Description

Add flash and reset instructions to the `README` of HS60 v2. It's important to erase the EEPROM, because of recent changes.

More information can be found here: https://github.com/qmk/qmk_firmware/pull/5193#issuecomment-466842609

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Wilba6582** [comment]
This might be redundant (or inaccurate) now that I've convinced @yiancar to enable VIA support in `via` keymaps only.

File: keyboards/hs60/v2/readme.md [hunk 1]
```diff
@@ -11,8 +11,27 @@ Hardware Availability: http://groupbuys.mechboards.co.uk/shop/hs60-hotswap-60-pc
...
+- Plug In
+- Unplug
+- Release Escape
+
+### Flash
+
+- Unplug
+- Hold Escape
+- Plug In
+- Flash using QMK Toolbox or dfu-util (`make hs60/v2:default:dfu-util`)
```
[the remark this answers is not in the document]
Comment (andys8) [reply keyboards/hs60/v2/readme.md]: Done :) You can squash commits on merge

**Comment by drashna** [comment]
@yiancar Merge this, or wait on a separate VIA keymap?

**Comment by andys8** [comment]
This PR adds currently working instructions. If there are upcoming changes, change the instructions and/or link to documentation, when this is the case ;)

**Comment by drashna** [comment]
@andys8 I do agree.  But if @yiancar has a PR for the VIA keymap in the next few days, I'd rather just wait on that.  Otherwise, yes, I'd like to merge this sooner rather than later.

**Comment by andys8** [comment]
Sounds good

**Comment by sevenseacat** [comment]
Can this be merged, we've just spent two hours headbanging wondering why the board wouldn't flash, because we didn't do the unplug/escape part....?

cc @lukerollans

**Pull request merged**